ArduPlane/Parameters.h

```diff
@@ -119,13 +119,13 @@ class Parameters {
         k_param_glide_slope_min,
         k_param_stab_pitch_down,
         k_param_terrain_lookahead,
-        k_param_fbwa_tdrag_chan,
+        k_param_fbwa_tdrag_chan, // unused - moved to RC option
         k_param_rangefinder_landing,
         k_param_land_flap_percent,  // unused - moved to AP_Landing
         k_param_takeoff_flap_percent,
         k_param_flap_slewrate,
         k_param_rtl_autoland,
-        k_param_override_channel,
+        k_param_override_channel, // unused
```

Comment:
Doesn't look unused to me?

Reply:
![image](https://user-images.githubusercontent.com/33176108/111006122-19667980-8384-11eb-8336-8dcd0c1ef473.png)

Unless I'm failing at VSCode.....

Reply:
Ah, I think i'm being caught out by the macro stuff, your right.